server/app/lib/fusor/validators/deployment_validator.rb

```diff
@@ -138,6 +156,19 @@ def validate_nfs_mount(deployment, address, path)
           return
         end
 
+        # 36 is the expected UID and GID of the share
+        if validate_nfs_uid(36, deployment.id) == false
```

Comment:
No particular reason other than to obfuscate the weird mount point name from the logic of the code. I can make that change though if you think it makes more sense just to put it there.